PendingReleaseNotes

```diff
@@ -1,5 +1,11 @@
 >=18.0.0
 
+* The PENDING_CREATING_PGS health check will have an additional warn message
+  "(PGs creation pending, temporarily increase 'osd_max_pg_per_osd_hard_ratio')" if
+  num_pgs on OSD is more than (osd_max_pg_per_osd_hard_ratio * mon_max_pg_per_osd).
+  It is recommended to add more OSDs to the cluster to resolve this condition.
+  Increase `osd_max_pg_per_osd_hard_ratio` to mitigate this warning temporarily.
+
```

Comment:
Suggest slight modification:
```
* The ``PENDING_CREATING_PGS`` health check will log an additional warning
  "(PG creation pending, temporarily increase 'osd_max_pg_per_osd_hard_ratio')" if
  the number of PGs on an OSD is greater than ``(osd_max_pg_per_osd_hard_ratio * mon_max_pg_per_osd).``
  It is recommended to add  OSDs to the cluster to resolve this condition.
  Increase `osd_max_pg_per_osd_hard_ratio` to mitigate this warning temporarily.

```

Also, below we say >= where here we say >, would be good to be consistent.